### PR TITLE
Adding review suggestions

### PR DESCRIPTION
**Pull request by Drifftr**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,3 +1,18 @@
+<<<<<<< HEAD
```
Comment (malakasilva) [inline pom.xml]: Seems like a conflict

File: src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java [hunk 2]
```diff
@@ -73,40 +90,51 @@ static private void addToZip(String path, String srcFile, ZipOutputStream zip) {
...
+	 */
 	static private void addFolderToZip(String path, String srcFolder, ZipOutputStream zip) {
 		File folder = new File(srcFolder);
 		String fileListArray[] = folder.list();
 		int i = 0;
 		if (fileListArray != null) {
 			while (i < fileListArray.length) {
 				String newPath = folder.getName();
-				if (!path.equalsIgnoreCase("")) {
+				if (!path.equalsIgnoreCase(HumanTaskPluginConstants.EMPTY_STRING)) {
```
Comment (malakasilva) [inline src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java]: You can use StringUtils for this purpose

File: src/main/java/org/wso2/maven/humantask/artifact/HumanTaskPluginConstants.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package org.wso2.maven.humantask.artifact;
```
Comment (cnapagoda) [inline src/main/java/org/wso2/maven/humantask/artifact/HumanTaskPluginConstants.java]: Missing License header

File: src/main/java/org/wso2/maven/humantask/artifact/HumanTaskPluginConstants.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+/**
+ * Created by heshitha on 9/5/16.
```
Comment (cnapagoda) [inline src/main/java/org/wso2/maven/humantask/artifact/HumanTaskPluginConstants.java]: ASF2.0 license, @author tags will no longer be allowed in any Apache source code. So please remove it.

File: pom.xml (hunk 1 above)
Comment (Drifftr) [inline pom.xml]: Thanks. Fixed in https://github.com/wso2/maven-tooling-bps/pull/8

File: src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java (hunk 2 above)
Comment (Drifftr) [inline src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java]: Thanks fixed in https://github.com/wso2/maven-tooling-bps/pull/8

File: src/main/java/org/wso2/maven/humantask/artifact/HumanTaskPluginConstants.java (hunk 3 above)
Comment (Drifftr) [inline src/main/java/org/wso2/maven/humantask/artifact/HumanTaskPluginConstants.java]: Thanks. Added license header in https://github.com/wso2/maven-tooling-bps/pull/8

File: src/main/java/org/wso2/maven/humantask/artifact/HumanTaskPluginConstants.java (hunk 4 above)
Comment (Drifftr) [inline src/main/java/org/wso2/maven/humantask/artifact/HumanTaskPluginConstants.java]: Thanks removed author tag in https://github.com/wso2/maven-tooling-bps/pull/8

File: src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java (hunk 2 above)
Comment (Drifftr) [inline src/main/java/org/wso2/maven/humantask/artifact/util/FileManagementUtils.java]: Thanks fixed in #8 and #9

**Pull request merged**